yjit/src/codegen.rs

```diff
@@ -3725,7 +3718,7 @@ fn gen_return_branch(cb: &mut CodeBlock, target0: CodePtr, target1: Option<CodeP
 
 fn gen_send_iseq(jit: &mut JITState, ctx: &mut Context, cb: &mut CodeBlock, ocb: &mut OutlinedCb, ci: * const rb_callinfo, cme: * const rb_callable_method_entry_t, block: Option<IseqPtr>, argc: i32) -> CodegenStatus
 {
-    let iseq = unsafe { def_iseq_ptr(get_cme_def(cme)) };
+    let iseq = unsafe { def_iseq_ptr((*cme).def) };
```

Comment:
If this function is a getter, maybe it should be named `get_def_iseq_ptr` ?

Reply:
Okay. I think that's the only one with that naming convention problem, though there are a few other inconsistencies of other types (e.g. functions in cruby.rs that could be moved over to bindgen.)